src/moments.jl

```diff
@@ -79,6 +79,29 @@ mean_and_std(A::RealArray, dim::Int) = (m = mean(A, dim); (m, stdm(A, m, dim)))
 mean_and_var(A::RealArray, wv::WeightVec, dim::Int) = (m = mean(A, wv, dim); (m, varm(A, wv, m, dim)))
 mean_and_std(A::RealArray, wv::WeightVec, dim::Int) = (m = mean(A, wv, dim); (m, stdm(A, wv, m, dim)))
 
+"""
+    mean_and_var(x::RealArray, wv::WeightVec, dim::Int)
+
+### Args:
+* `x`: A vector of type `RealArray` to compute theh mean and variance of.
+* `wv`: An optional weight vector of type `WeightVec`.
+* `dim`: An optional `Int` type parameter which specifies the dimension.
+
+Jointly compute the mean and variance of x.
+"""
+function mean_and_var end
```

Comment:
Why not add to docstring to the actual definition above? Same for `mean_and_std`, and (I think) in other places too.